### PR TITLE
Add Messenger icon

### PR DESCRIPTION
**Pull request by samhpham**
## ❓ Context
For adding social sharing buttons to Comparison product and Messenger is one of the platforms we would like to support.

## 🚀 Changes
Add Messenger SVG

## ✅ Checklist

- [x] The package version is bumped according to README in `package.json`, `package-lock.json`, and `CHANGELOG.md`

### CODE REVIEW

File: icons/messenger.svg [hunk 1]
```diff
@@ -0,0 +1 @@
+<svg id="artwork" xmlns="http://www.w3.org/2000/svg" viewBox="0 0 24 24"><title>messenger</title><path d="M12 0C5.373 0 0 4.975 0 11.111C0 14.608 1.745 17.727 4.472 19.763V24L8.558 21.758C9.648 22.059 10.804 22.222 12 22.222C18.627 22.222 24 17.248 24 11.111C24 4.975 18.627 0 12 0V0ZM13.193 14.963L10.137 11.704L4.174 14.963L10.733 8L13.863 11.259L19.752 8L13.193 14.963Z"/></svg>
```
Comment (dozortsev) [inline icons/messenger.svg]: could you please call it `facebook-messenger.svg`

File: CHANGELOG.md [hunk 2]
```diff
@@ -1,3 +1,6 @@
+# v1.5.0
+## Adds `facebook-messenger` icon
```
Comment (oliverviljamaa) [inline CHANGELOG.md]: Could it just be `messenger`?

File: icons/facebook-messenger.svg [hunk 3]
```diff
@@ -0,0 +1 @@
+<svg id="artwork" xmlns="http://www.w3.org/2000/svg" viewBox="0 0 24 24"><title>facebook-messenger</title><path d="M12 0C5.373 0 0 4.975 0 11.111C0 14.608 1.745 17.727 4.472 19.763V24L8.558 21.758C9.648 22.059 10.804 22.222 12 22.222C18.627 22.222 24 17.248 24 11.111C24 4.975 18.627 0 12 0V0ZM13.193 14.963L10.137 11.704L4.174 14.963L10.733 8L13.863 11.259L19.752 8L13.193 14.963Z"/></svg>
```
Comment (oliverviljamaa) [inline icons/facebook-messenger.svg]: Can we try minifying it a bit on:
https://jakearchibald.github.io/svgomg/ (precision 1 is usually a good match)

File: CHANGELOG.md (hunk 2 above)
Comment (oliverviljamaa) [reply CHANGELOG.md]: It's a Facebook product, but standalone, with an URL:
https://messenger.com

Comment (dozortsev) [reply CHANGELOG.md]: lol, I asked use this name https://github.com/transferwise/icons/pull/18#discussion_r344730520

Comment (dozortsev) [reply CHANGELOG.md]: IMO Facebook Messenger is better 🙂

File: icons/facebook-messenger.svg (hunk 3 above)
Comment (dozortsev) [reply icons/facebook-messenger.svg]: I often see people say about this in PRs for this repo, time add this in README.md?

File: CHANGELOG.md (hunk 2 above)
Comment (samhpham) [reply CHANGELOG.md]: @dozortsev I slightly do agree more with Oliver since 'Messenger' is the product name? But what do you think?

Comment (dozortsev) [reply CHANGELOG.md]: I'm not so cool as you and I thought that this is confusing name, messenger is very vague name for facebook messenger

**Pull request merged**